### DISCUSSION TITLE
Code Mark (inline) syntax highlighting

### DISCUSSION THREAD

**Discussion by bjornhansen** [ueberdosis/tiptap#2574] [Feature Requests]
I would love for there to be an option for the [Code Mark](https://tiptap.dev/api/marks/code) to have a syntax highlighting option similar to the [CodeBlockLowlight Node](https://tiptap.dev/api/nodes/code-block-lowlight).

Is this possible or in the works? Or are there easy workarounds that anyone knows about?

**Comment by cadars** [comment]
I would create a custom CodeBlock extension, with an extra class to style `<pre><code>…</code></pre>` inline in CSS?

**Reply by bjornhansen** [reply]
Thanks, I'll give that a try.

**Comment by nartix** [comment]
I've created a solution if you're still interested! I've ported the @tiptap/extension-code-block-lowlight extension to work with Code Mark. Here's how to install it:
`npm install @nartix/tiptap-inline-code-highlight`
